src/components/deposit/Confirming.js

```diff
@@ -45,9 +38,10 @@ const Confirming = ({ signerFeeInSatoshis, error }) => {
 }
 
 const mapStateToProps = (state) => {
+  const { signerFeeInSatoshis, btcConfirmingError } = state.deposit
```

Comment:
Tbh if we're doing this I feel like we might as well decompose in the function parameters directly.